tests/scripts/create-and-setup-gcp-vm.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+PARENT_DIR=$(dirname "$SCRIPT_DIR")
+
+source scripts/vars.env
+
+gcloud compute firewall-rules create ${FIREWALL_RULE_NAME} \
+    --project=${GKE_PROJECT} \
+    --direction=INGRESS \
+    --priority=1000 \
+    --network=default \
+    --action=ALLOW \
+    --rules=tcp:22 \
+    --source-ranges=$(curl -sS -4 icanhazip.com)/32 \
```

Comment:
I'm a bit concerned that use some not-well-known external servics, especially from security perspective. 
would it be too difficult if we make the source range configurable in the vars file?